app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None:
+        if "vaccine" not in visitor:
```

Comment:
What if it's none? Use get function

Reply:
Using `.get` would be more efficient here